examples/multi-tenancy-with-teams/main.tf

```diff
@@ -141,14 +141,13 @@ module "aws-eks-accelerator-for-terraform" {
   }
 
   platform_teams = {
-    admin-team-1 = {
+    admin = {
       ## Users Example:
       # users = [
       #   "arn:aws:iam::<ACCOUNT_ID>:user/<USERNAME>",
       #   "arn:aws:iam::<ACCOUNT_ID>:role/<ROLE_NAME>"
       # ]
```

Comment:
Can we remove this commented code or explain why we commented?

Reply:
I have been asked by few consultants on what this commented section is about. Do we need to enable etc.? May be we can add some wording to support that